### ISSUE TITLE
The output of trained model is constant when using batch normalization

### ISSUE THREAD

**Issue by LJXLJXLJX** [tensorflow/tensorflow#23630]
I have trained a model with tensorflow, with the use of batch normalization.

```
conv2 = slim.conv2d(
inputs=pool1,
num_outputs=64,
kernel_size=[5, 5],
padding="same",
normalizer_fn=slim.batch_norm,
activation_fn=tf.nn.relu)
```
And the model has been trained well, I can see the loss curve has been falling in tensorboard, and the validation accuracy reached 99%+

Then I saved my model as ckpt files. But when I tried to load my model from ckpt files and do some prediction, what ever I feed to the model, the output is always an constant array.

Then I removed all bn layer, and train my model and save it as ckpt again.When I load it and do prediction, it works well.

Can anybody tell me why?
